.azure-pipelines/docker.yml

```diff
@@ -1,19 +1,176 @@
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+ 
 trigger:
   branches:
     include:
-      - 'task/add-acr-pipeline'
-  tags:
-    include:
-      - v*
-schedules:
-  - cron: "0 18 * * 4"
-    displayName: "Weekly Publish Docker Image"
-    branches:
-      include:
-        - main
-    always: true
-
+    - main
+    - task/add-new-acr-pipeline
+ 
+pr: none
+ 
 variables:
-  REGISTRY: "msgraphprod.azurecr.io"
-  IMAGE_NAME: "public/openapi/kiota"
-  PREVIEW_BRANCH: "refs/heads/main"
\ No newline at end of file
+  buildConfiguration: 'Release'

```

Comment:
rather than adding a new pipeline, can this be another stage for the existing one? this way it's all coordinated

Reply:
Ok. Sounds good. Let me update the current pipeline